seed/analysis_pipelines/bsyncr.py

```diff
@@ -108,74 +109,62 @@ def _prepare_all_properties(analysis_property_view_ids, analysis_id, progress_da
     analysis_property_views = AnalysisPropertyView.objects.filter(id__in=analysis_property_view_ids)
     input_file_paths = []
     for analysis_property_view in analysis_property_views:
-        try:
-            meters = (
```

Comment:
I removed the `try/except` clauses which previously caught unexpected errors (now handled by the `analysis_pipeline_task` decorator). Kind of a bummer it makes it look like everything was modified, sorry ☹️

Reply:
Thanks for pointing it out - were there any changes at all other than removing the try-except blocks within these lines 111 - 155?

Reply:
No, there weren't any other changes in that range